feat!: Replace Aspect Ratio component with utility classes and prop on Image (#1593)
Amsterdam/design-system: packages/css/src/components/image/README.md

Displays an image.

## Design

> Every Image must display in one of the [available aspect ratios](https://designsystem.amsterdam/?path=/docs/brand-design-tokens-aspect-ratio--docs).

Reviewer: ```suggestion
Every Image should be shown in one of the [available aspect ratios](https://designsystem.amsterdam/?path=/docs/brand-design-tokens-aspect-ratio--docs).
```

The default is 16:9.

> If the intrinsic dimensions of the source do not match an aspect ratio, the image will get cropped to cover the intended area without distorting its appearance.

Reviewer: ```suggestion
If the intrinsic dimensions of the source do not match an aspect ratio, the image will get cropped to cover the intended area.
```

  For example, provide small, medium, and large variants for various screen sizes.
  This prevents unnecessary downloading of large files.
  Do this especially for the main image of a page, where the difference between sizes on a narrow and wide screen is most significant.

> - Ensure that the aspect ratio of the image is supported by the [aspect ratio token](/docs/brand-design-tokens-aspect-ratio--docs). Alternatively, use the `cover` Image prop or the `object-fit: contain` CSS property to display the image correctly.

Reviewer 2: Something like this? (Following from ideas in other comments.)

```suggestion
- The image has an aspect ratio of 16:9 by default. [Other options](/docs/brand-design-tokens-aspect-ratio--docs) are available.
```
